persistence-model/src/main/scala/hmda/persistence/messages/events/processing/HmdaFileParserEvents.scala

```diff
@@ -0,0 +1,14 @@
+package hmda.persistence.messages.events.processing
+
+import hmda.model.fi.lar.LoanApplicationRegister
+import hmda.model.fi.ts.TransmittalSheet
+import hmda.model.parser.LarParsingError
+import hmda.persistence.messages.CommonMessages.Event
+
+object HmdaFileParserEvents {
+  trait HmdaFileParserEvent extends Event
```

Comment:
@jmarin I saw this pattern in several of your serializer PRs, so I followed it. What's the benefit of using this specialized event trait?